index.html

```diff
@@ -75,6 +75,270 @@ <h1 class="headline-first-title">I'm Emmanuel Ikpe <br />
+                                    fill="#505F79" />
+                            </svg>
+                        </a></li>
+                </ul>
+                <button type="button" class="see-more">Get my resume</button>
+            </div>
+            <div>
+                <div class="language-section">
+                    <div class="dropdown-menu">
+                        <p class="lang-text">Langguages</p>
+                        <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
+                            <path fill-rule="evenodd" d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z" clip-rule="evenodd" />
+                          </svg>
+                    </div>
+                    <div class="language">
+                        <div class="lang">
+                            <img alt="js" src="/assets/js.png">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="lang">
+                            <img alt="html" src="/assets/html.png">
+                            <p>HTML</p>
+                        </div>
+                        <div class="lang">
+                            <img alt="css" src="/assets/css.png">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] Please use a `ul` and a `li` for each technology. The reason is that this is a list of technologies.

See example:
```html
<ul>
    <li>tech 1</li>
    <li>tech 2</li>
</ul>
```